experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+  && apt-get install --yes \
+    openjdk-8-jdk \
+    iproute2 \
+    bash \
+    sudo \
+    coreutils \
+    procps \
+    strace \
```

Comment:
why do we need strace? this is different from before.

Reply:
Strace is needed for the parameter that can be set to activate strace: spark.databricks.daemon.driver.python.withStrace

Reply:
And README.md created for 7.x

Reply:
It seems like `withStrace` is false by default. I'd leave this commented out with a note saying that strace should be installed for users who are trying to set `withStrace` on their clusters

Reply:
where's the README?

Reply:
It is included in a separate pull request.

Reply:
@nadroj94 please leave a comment for strace and we can merge this. thanks!